elixir/config/runtime.exs

```diff
@@ -213,4 +213,8 @@ if config_env() == :prod do
            adapter: compile_config!(:outbound_email_adapter),
            from_email: compile_config!(:outbound_email_from)
          ] ++ compile_config!(:outbound_email_adapter_opts)
+
```

Comment:
Just double-checking here -- what was the reasoning behind not using `compile_config!`?

Reply:
Good catch. Yeah, we must use `compile_config!/1` here too.

Reply:
Ah, yeah, I think this was just a "it's working!" thing and I forgot to come back and update this.  Will get this updated to use `compile_config!/1`